conjure-lib/src/main/java/com/palantir/conjure/java/lib/internal/ConjureCollectionType.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.lib.internal;
+
+public enum ConjureCollectionType {
+    LIST("List"),
+    SET("Set");
+
+    private final String collectionName;
+
+    ConjureCollectionType(String collectionName) {
+        this.collectionName = collectionName;
+    }
+
+    public String getCollectionName() {
+        return collectionName;
+    }
+}
```

Comment:
I don't think this type should exist -- conjure-lib is the core API required by our generated code, and this seems like more of a utility for codegen. Codegen should be largely self-contained

Reply:
that makes sense. moved to `com.palantir.conjure.java.util`